test/jdl/grammar/grammar.spec.js

```diff
@@ -1081,24 +1086,31 @@ entity A {
 
       it('should parse it', () => {
         expect(parsedEnum).to.deep.equal({
+          javadoc: ' country enum ',
```

Comment:
I find it weird the comment isn't trimmed, WDYT? I would have expected it to be `country enum`.

Reply:
I tried to keep it in sync with how entity comments are being parsed https://github.com/jhipster/generator-jhipster/blob/87a393defaee9acfd25e05d78c00440878445392/test/jdl/grammar/grammar.spec.js#L568 https://github.com/jhipster/generator-jhipster/blob/87a393defaee9acfd25e05d78c00440878445392/test/jdl/grammar/grammar.spec.js#L521

But I agree there is no use of trailing space, I have now fixed this for entities as well by changing regex in `jdl-ast-builder-visitor.js` `trimComment()`